boot/zk_common.sh

```diff
@@ -56,7 +56,8 @@ function zk_common_delegated_dataset
     fi
 
     chmod 777 ${ZK_ROOT}
-    sudo -u zookeeper mkdir -p ${ZK_ROOT}/zookeeper
+    mkdir -p ${ZK_ROOT}/zookeeper
+    chown zookeeper:zookeeper ${ZK_ROOT}/zookeeper
```

Comment:
What happens here on reprovision of a pre-existing binder zone that might have files under `${ZK_ROOT}/zookeeper` owned by the "hadoop" group -- assuming I have that correct?  Need that be a `chown -R ...`?

Reply:
Good call, will fix that.